### DISCUSSION TITLE
Figma UX/UI Designs

### DISCUSSION THREAD

**Discussion by umitcan07** [bounswe/bounswe2024group11#190] [Show and tell]
[Link](https://www.figma.com/file/BefZiYhQw14Bo1euCuhyBh/Mockups?type=design&node-id=7%3A2218&mode=design&t=A6WSbZ9B9MtcTqbM-1) to our Figma file

**Comment by Meminseeker** [comment]
I put a review on the design #2 on the scenario #2 with a huge and colorful text


Please disregard the links on the hashtags; I just meant `the number`.
